src/parking/controls/editor/editor-form.ts

```diff
@@ -89,6 +90,7 @@ const parkingLaneTagTemplates = [
     'parking:lane:{side}',
     'parking:lane:{side}:{type}',
     'parking:condition:{side}',
+    'parking:condition:{side}:conditional',
```

Comment:
Should this also include `maxstay:conditional` for when stay duration is limited only during the day for example

Also, should `parking:condition:{side}:time_interval` or `parking:condition:{side}:default'` be displayed any more? Or should they still be displayed in case there is legacy data?

Reply:
> Or should they still be displayed in case there is legacy data?`parking:condition:{side}:time_interval``parking:condition:{side}:default'`

yes

Reply:
> Should this also include `maxstay:conditional` for when stay duration is limited only during the day for example

Will need to be added in the future